### ISSUE TITLE
Better light-theme support

### ISSUE THREAD

**Issue by kjy5** [AtomMaterialUI/a-file-icon-idea#458]
**Is your feature request related to a problem? Please describe.**
The icons look fantastic against a dark background, however they have low visibility on a light one.

**Describe the solution you'd like**
Some alternate set or value change that is sensitive to whether a light or dark theme is used.

**Describe alternatives you've considered**
Just simply a way to change the overall value (brightness/darkness) of icons

**Additional context**
<img width="39" alt="Screenshot 2024-06-12 at 8 50 15 AM" src="https://github.com/mallowigi/a-file-icon-idea/assets/82800265/52fef6f8-fe48-4688-897d-eb83c87b3fe4">

The structure and three dots have low contrast against the default light theme

**Comment by mallowigi** [comment]
JetBrains has a way of specifying icon variants, but this is not something I'm paying a lot of attention since I'm not using light themes.

If you have other icons that you think would require a different color, please post it on this thread, I'll issue variations for these icons.

**Comment by kjy5** [comment]
Mostly the yellow and green icons have low contrast against a lighter background, particularly if they have thin strokes. Is this related to the "Saturated Icons" option? That button doesn't seem to do anything for me, or I can't tell if something happened with the light theme. **I think a simple solution where a button can be added to universally darken all icons will help.** Even better if it can be tied to the preferred theme (light/dark) switch so it happens automatically.

I've been using your dark themes and this icons pack together for years and it's been great! I've only recently started experimenting with automatically switching between dark and light themes depending on my ambient lighting conditions which is how I noticed the issue.
